src/Service/LeantimeApiService.php

```diff
@@ -67,6 +74,11 @@ public function getAllProjects(): mixed
         return $this->request(self::API_PATH_JSONRPC, 'POST', 'leantime.rpc.projects.getAll', []);
     }
 
+    private function getProjectTicketStatusSettings(string $projectId): mixed
```

Comment:
Can this return an array instead of mixed? That would clean up the other parts of the code

Reply:
Api returns stdClass by default, so i changed the return type from mixed to stdClass. 
Why should it be converted to array? And do you still think it should?

Reply:
The api returns stdClass by default. I changed the return type from mixed to stdClass. 
Do you want me to convert it to an array?